tsl/src/continuous_aggs/invalidation.h

```diff
@@ -37,7 +36,7 @@ typedef struct Hypertable Hypertable;
 
 extern void invalidation_cagg_log_add_entry(int32 cagg_hyper_id, int64 modtime, int64 start,
 											int64 end);
-extern void invalidation_hyper_log_add_entry(int32 hyper_id, int64 modtime, int64 start, int64 end);
+extern void invalidation_hyper_log_add_entry(int32 hyper_id, int64 start, int64 end);
```

Comment:
The function above, `invalidation_cagg_log_add_entry` still takes `modtime`.

Reply:
Removed (and all resulting undefined code).